pubsub/src/subscriber.rs

```diff
@@ -314,7 +314,7 @@ mod tests {
             PublishRequest {
                 topic: "projects/local-project/topics/test-topic1".to_string(),
                 messages: vec![PubsubMessage {
-                    data: "hoge".as_bytes().to_vec(),
+                    data: Bytes::from_static(b"hoge"),
```

Comment:
Since it is hard to modify all the test code according to the google-cloud-googleapis feature, how about `"hoge".into();` so that it can be used for both `Vec<u8>` and `Bytes`?

Reply:
Thanks for the suggestion. I'll put some more effort into properly implementing this if you're okay with adding it as a new feature.